spec/features/users/sign_up_spec.rb

```diff
@@ -14,6 +14,52 @@
     end
   end
 
+  context 'and set up voice as 2FA' do
```

Comment:
How would you feel about moving these into their own spec file in the `backup_mfa` folder? Something like `otp_delivery_selection_spec.rb`?

Reply:
Moved to separate spec file.